coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -756,15 +780,15 @@ public Response leaveLRA(
             @HeaderParam(HttpHeaders.ACCEPT) @DefaultValue(MediaType.TEXT_PLAIN) String mediaType,
             @Parameter(ref = LRAConstants.NARAYANA_LRA_API_VERSION_HEADER_NAME)
             @HeaderParam(LRAConstants.NARAYANA_LRA_API_VERSION_HEADER_NAME) @DefaultValue(CURRENT_API_VERSION_STRING) String version,
-            String participantCompensatorUrl) throws NotFoundException {
+            String participantCompensatorUrl) {
```

Comment:
Hmm, public API change but I guess it might not break anyone that was coding against it as I think the exception being a RuntimeException won't result in a compiler error - do you agree?

Reply:
I agree: NotFoundException is a subclass of WebApplicationException and is a convenience - the equivalent is `WebApplicationException.getResponse().getStatus() == NOT_FOUND`.